src/System.Buffers/src/System.Buffers.csproj

```diff
@@ -6,18 +6,23 @@
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
     <DocumentationFile>$(OutputPath)$(AssemblyName).xml</DocumentationFile>
     <UseOpenKey Condition="'$(UseOpenKey)'==''">true</UseOpenKey>
-    <NuGetTargetMoniker Condition="'$(TargetGroup)' == ''">.NETStandard,Version=v1.1</NuGetTargetMoniker>
+    <IsPartialFacadeAssembly Condition="'$(TargetGroup)' == '' or '$(TargetGroup)' == 'netcoreapp1.1'">true</IsPartialFacadeAssembly>
+    <NuGetTargetMoniker Condition="'$(TargetGroup)' == ''">.NETCoreApp,Version=v1.1</NuGetTargetMoniker>
+    <PackageTargetFramework Condition="'$(TargetGroup)' == ''">netcoreapp1.1</PackageTargetFramework>
```

Comment:
This is `">.NETStandard,Version=v1.7` everywhere else. e.g. https://github.com/dotnet/corefx/blob/master/src/System.Runtime/src/System.Runtime.csproj#L12 . Any reason for it to be different?